osbs/api.py

```diff
@@ -589,6 +589,18 @@ def get_image_stream_tag(self, tag_id):
         return self.os.get_image_stream_tag(tag_id)
 
     @osbsapi
+    def put_image_stream_tag(self, tag_id, tag):
```

Comment:
What will use this API method? I see why the ensure method would be used, but I'm not sure I see why this method would be used on its own.